app/components/Setting.vue

```diff
@@ -444,7 +444,17 @@ export default {
                     // let blks = await contract.getWithdrawBlockNumbers.call({ from: account })
 
                     const blks = await blksPromise
-                    await Promise.all(blks.map(async (it, index) => {
+
+                    // remove duplicate
+                    const set = new Set()
+                    for (let i = 0; i < blks.length; i++) {
+                        if (!set.has(blks[i])) {
+                            set.add(blks[i])
+                        }
+                    }
+                    let blks2 = Array.from(set)
```

Comment:
Should use lodash lib (uniqBy). See https://stackoverflow.com/questions/31740155/lodash-remove-duplicates-from-array
just 3-line code.

Reply:
I made a comparison between lodash uniqBy and Set()
![image](https://user-images.githubusercontent.com/24842503/56550395-b90f9880-65af-11e9-8f15-2eae9be96e68.png)
I will stick with Set() but will update my code

Reply:
:), yep, your code is faster, but trade-off.